### DISCUSSION TITLE
Can an option to convert to DV profile 7.6 be added?

### DISCUSSION THREAD

**Discussion by MarkPeloquin** [quietvoid/dovi_tool#76] [Ideas]
Doing some reencoding of a number of DV videos that have DV profiles of 4,5,6,7. Plex doesn't like Profile 6 for sure, but until I try my conversion script on 4 & 5, won't be sure about them. Would it be possible to have a "mode" option that converts Profiles to 7.6?

TIA

**Comment by quietvoid** [comment]
You seem to be confusing the metadata in the container with the one in the bitstream.
There's no such thing as profile 7.6.

**Reply by MarkPeloquin** [reply]
Apologies for confusing enumerations. I was hoping to have a conversion option that would result in (from mediainfo): "Dolby Vision, Version 1.0, dvhe.07.06, BL+EL+RPU". Profile 7 works flawlessly in Plex. Thanks

**Reply by quietvoid** [reply]
There's no conversion to profile 7 unless it's MEL, then you can use mode 1 with existing profile 8 RPUs.
Not possible from the others you mention.

**Comment by ghost** [comment]
I'm personally rather interested in Profile 9, for conversion of bluray to DoVi without doing transcode, the showstopper there would be that MadVR can't analyze SDR material to get usuable L1 out of.  Fixed NPL conversion in ffmpeg / zscale to X265 -> ST2084 -> MadVR -> generate -> inject. with no grading does a suprisingly good job on newer movies with less grain but Converting to 10-bit PQ HEVC is just incredibly slow with the settings needed to make it look good (--aq-mode 4 really improves things but really tanks performance regardless of x265 profile), then that needs to follow up with a (faster) analysis.  

I've never seen this profile used in the wild so I doubt MadVR alone would suffice (and don't really know what it should look like), and injecting metadata into H.264 would then become necessary, but it's something interesting to consider if it's someow doable and isn't deprecated.

**Comment by ghost** [comment]
I said it, didn't I?   Conversion of regular BD to HDR without 9-15 hours
of reencoding on slow profile x265.
